transports/quic/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "libp2p-quic"
+version = "0.8.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+if-watch = "2.0.0"
+libp2p-core = { version = "0.37.0", path = "../../core" }
+libp2p-tls = { version = "0.1.0-alpha", path = "../../transports/tls" }
+quinn = { version = "0.9.0", git = "https://github.com/quinn-rs/quinn.git", branch = "main", features = ["tls-rustls", "futures-io", "runtime-async-std"] }
+futures = "0.3.21"
+thiserror = "1.0.26"
+tracing = "0.1"
+rand = "0.8.5"
+
+oid-registry = "0.6.0"
+webpki = "0.22.0"
+rcgen = "0.9.2"
+ring = "0.16.20"
+rustls = { version = "0.20.2", default-features = false, features = ["dangerous_configuration"] }
+x509-parser = "0.14.0"
+yasna = "0.5.0"
```

Comment:
I think some of these could be dropped now that you are depending on `libp2p-tls`.